src/Aspire.Hosting.Keycloak/Aspire.Hosting.Keycloak.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
```

Comment:
We should add SuppressFinalPackageVersion to both this and Aspire.Keycloak.Authentication libraries. I've done this in https://github.com/julioct/aspire/pull/1 if you want to merge that change into this PR.

Reply:
Merged your PR, thank you!